src/utils/formatting.ts

```diff
@@ -3,6 +3,7 @@ import { Bet } from "../types/entities";
 
 export const formatToFourDecimals = (amount: string) => {
   const parsedAmount = parseFloat(amount);
+  if (parsedAmount === 0) return "0";
```

Comment:
Good catch.  May aswell update the unit test for this condition too?

Reply:
will do 👍🏻